implants/lib/eldritch/src/runtime/messages/report_error.rs

```diff
@@ -0,0 +1,27 @@
+use super::{Dispatcher, Transport};
+use anyhow::Result;
+use pb::c2::{ReportTaskOutputRequest, TaskError, TaskOutput};
+
+#[cfg_attr(debug_assertions, derive(Debug))]
+#[derive(Clone)]
+pub struct ReportErrorMessage {
+    pub(crate) id: i64,
+    pub error: String,
+}
+
+impl Dispatcher for ReportErrorMessage {
+    async fn dispatch(self, transport: &mut impl Transport) -> Result<()> {
+        transport
+            .report_task_output(ReportTaskOutputRequest {
+                output: Some(TaskOutput {
+                    id: self.id,
+                    output: String::from(""),
+                    exec_started_at: None,
+                    exec_finished_at: None,
+                    error: Some(TaskError { msg: self.error }),
+                }),
+            })
+            .await?;
```

Comment:
i think for these you can just remove `?` and the `Ok(())` as its the same thing.

Reply:
They return the responses